elephantblog/management/commands/blogping.py

```diff
@@ -69,8 +69,8 @@ def handle_noargs(self, **options):
                     create_kwargs = {
                         'content_object': entry,
                         'weblogname': site.name,
-                        'weblogurl': site.domain + ':/' + PINGING_WEBLOG_URL,
-                        'changesurl': site.domain + ':/' + entry.get_absolute_url(),  # noqa
+                        'weblogurl': 'http://' + site.domain + PINGING_WEBLOG_URL,
```

Comment:
Vorschlag: PINGING_WEBLOG_URL muss immer in der selben Form sein wie die absolute_url 

Es gibt ja auch noch die Variable PINGING_WEBLOG_NAME, welche für den Name verwendet wird.

Was meinst du dazu?


            if self.use_sites:
                for site in entry.sites.all():
                    create_kwargs = {
                        'content_object': entry,
                        'weblogname': site.name,
                        'weblogurl': 'http://' <http://'> + site.domain + PINGING_WEBLOG_URL,
                        'changesurl': 'http://' <http://'> + site.domain + entry.get_absolute_url(),  # noqa
                    }
                    if not options.get('dryrun'):
                        PingedURL.objects.create_for_servers(**create_kwargs)
            else:
                create_kwargs = {
                    'content_object': entry,
                    'weblogname': PINGING_WEBLOG_NAME,
                    'weblogurl': http://' <http://'> + domain + PINGING_WEBLOG_URL,
                    'changesurl': 'http://' <http://'> + domain + entry.get_absolute_url(),
                }

> Am 25.02.2015 um 11:13 schrieb Simon Bächler <notifications@github.com <mailto:notifications@github.com>>:
> 
> In elephantblog/management/commands/blogping.py <https://github.com/feincms/feincms-elephantblog/pull/48#discussion_r25329208>:
> 
> > @@ -69,8 +69,8 @@ def handle_noargs(self, **options):
> >                      create_kwargs = {
> >                          'content_object': entry,
> >                          'weblogname': site.name,
> > -                        'weblogurl': site.domain + ':/' + PINGING_WEBLOG_URL,
> > -                        'changesurl': site.domain + ':/' + entry.get_absolute_url(),  # noqa
> > +                        'weblogurl': 'http://' <http://'> + site.domain + PINGING_WEBLOG_URL,
> This does not make sense yet. PINGING_WEBLOG_URL can be an URL or a name. Can you fix this as well?
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/feincms/feincms-elephantblog/pull/48/files#r25329208>.
>